### DISCUSSION TITLE
How to load native so file in android

### DISCUSSION THREAD

**Discussion by jeff9571** [AvaloniaUI/Avalonia#12243] [Q&A]
I'm trying to use ffmpeg lib in android, but I don't know where should I put the so files and how to load them.
![image](https://github.com/AvaloniaUI/Avalonia/assets/35749706/be7ebd71-9171-4539-9a3d-ef29ce62aca3)
I want to do something like this [https://docs.avaloniaui.net/docs/next/guides/platforms/macos-development#native-code](url)

**Comment by maxkatz6** [comment]
https://learn.microsoft.com/en-us/xamarin/android/platform/native-libraries
